tests/components/modbus/test_light.py

```diff
@@ -217,7 +223,18 @@ async def test_all_light(hass: HomeAssistant, mock_do_cycle, expected) -> None:
 
 @pytest.mark.parametrize(
     "mock_test_state",
-    [(State(ENTITY_ID, STATE_ON),)],
+    [
```

Comment:
I think we should set config for 2 lights (a simple one and one with temp+brightness), and have different states for the 2 entities that get created for each config